test/wcsmo14/demos/truss/truss_compliance_2d2.jl

```diff
@@ -44,10 +44,10 @@ TopOpt.setpenalty!(solver, p)
 
 @show obj(r.minimizer)
 @show constr(r.minimizer)
-# fig = visualize(
-#     problem; u=solver.u, topology = r.minimizer,
-#     default_exagg_scale=0.0
-# )
-# Makie.display(fig)
+fig = visualize(
+    problem, solver.u, topology = r.minimizer,
```

Comment:
This part. 

The code fails to recognize u as one of the argument, and a semi-colon was used after problem

Remove them would make the script run smoothly.

Reply:
@yijiangh is this a recent change?

Reply:
@pitipatw the visualizer should be able to pick up `u` by calling `fig = visualize(problem; solver.u, topology=minimizer)`. The related source code can be found [here](https://github.com/JuliaTopOpt/TopOpt.jl/blob/5c1c74a9662557b4a4a2f3c8197c7a398fc4d081/src/TopOptProblems/Visualization/makie.jl#L175-L176). Would you mind trying again?

BTW, the youtube video is a bit outdated for our current master branch. Sorry for that.

Reply:
I tried fig = visualized(problem; solver.u, topology=minimizer) and get error :MethodError: no method matching visualize(...

Currently, I can run fig = visualize(problem , solver.u ... without error

only the comma (,) and semi-colon (;) that is different.

Reply:
Ah! You are using v0.6.1, and I did clean up the visualization code to make `u` appears after the semicolon `;`, change can be found [here](https://github.com/JuliaTopOpt/TopOpt.jl/compare/v0.6.1...master#diff-c012c87b78f02b909b1b32a8623ad5269a7ac1ef08c502917b7cba4caab53ebeL20-L23).
If you pull the master branch, you should see that problem go away!

@mohamed82008 shall we do a minor version bump to v0.6.2 for the viz?

Reply:
let's release 0.7

Reply:
I see.